apps/cli/src/sunodoCommand.ts

```diff
@@ -1,7 +1,26 @@
-import { Command, Config } from "@oclif/core";
+import { Command, Interfaces } from "@oclif/core";
+import { BuildStore } from "./utils/store";
 
-export abstract class SunodoCommand extends Command {
-    protected constructor(argv: string[], config: Config) {
-        super(argv, config);
+export type Flags<T extends typeof Command> = Interfaces.InferredFlags<
+    (typeof SunodoCommand)["baseFlags"] & T["flags"]
+>;
+export type Args<T extends typeof Command> = Interfaces.InferredArgs<T["args"]>;
+
+export abstract class SunodoCommand<T extends typeof Command> extends Command {
+    protected flags!: Flags<T>;
+    protected args!: Args<T>;
+
+    protected buildStore: BuildStore = new BuildStore(process.cwd());
```

Comment:
Can you explain how is this being used?

Reply:
I forgot to delete it. I was going to write a manager for the build artifacts, but then I decided to simplify and keep just one artifact.